scripts/validate_schemas.sh

```
@@ -47,55 +56,59 @@ for schema in "${!EXAMPLES[@]}"; do
 echo "  - ajv validate: examples/v1/policy/governance_min.json against schemas/v1/policy/governance_policy.schema.json"
-ajv validate "${AJV_BASE_ARGS[@]}" -s schemas/v1/policy/governance_policy.schema.json -d examples/v1/policy/governance_min.json
+run_ajv validate -s schemas/v1/policy/governance_policy.schema.json -d examples/v1/policy/governance_min.json
 
 echo "[schemas] Additional encoding tests (ed25519 base64url forms)…"
-# Root schemas that reference defs using the canonical $id for proper resolution
-printf '{"$schema":"https://json-schema.org/draft/2020-12/schema","$ref":"https://gatos.dev/schemas/v1/common/ids.schema.json#/$defs/ed25519Key"}' > /tmp/ed25519Key.schema.json
-printf '{"$schema":"https://json-schema.org/draft/2020-12/schema","$ref":"https://gatos.dev/schemas/v1/common/ids.schema.json#/$defs/ed25519Sig"}' > /tmp/ed25519Sig.schema.json
+# Create temporary schemas within the repository workdir so the container can access them via the bind mount
+TMPDIR_HOST="$(mktemp -d -p "$PWD" .ajvtmp.XXXXXX)"
+TMPDIR_REL="$(basename "$TMPDIR_HOST")"
+printf '{"$schema":"https://json-schema.org/draft/2020-12/schema","$ref":"https://gatos.dev/schemas/v1/common/ids.schema.json#/$defs/ed25519Key"}' > "$TMPDIR_HOST/ed25519Key.schema.json"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)